domain/user/state/state.go

```diff
@@ -247,7 +249,7 @@ SELECT (
        ) AS (&User.*)
 FROM user
 JOIN user_password ON user.uuid = user_password.user_uuid
-WHERE user.uuid = $M.uuid AND removed = false
+WHERE user.name = $M.name AND removed = false
 `
 
 		selectGetUserWithAuthStmt, err := sqlair.Prepare(getUserWithAuthQuery, User{}, sqlair.M{})
```

Comment:
There's only one query and statement in this method. Just use short names.

Looking at the query itself above, you should be able to select `User.*` directly without explicitly listing the columns.

Reply:
Without it, we faced problems of finding "disabled" and "last_login" columns, which are not a part of this JOIN.

Reply:
Discussed on the meeting. Decided to stay without changes.